### PR TITLE
fix(minor): [sc-9450] Add possibility to list consumer groups.

### PR DESCRIPTION
**Pull request by ser-0xff**
## Description

Add possibility to list consumer groups.

## How Has This Been Tested?

Unit tests.

## Minimal checklist:

- [X] I have performed a self-review of my own code 
- [ ] I have added `DocC` code-level documentation for any public interfaces exported by the package
- [ ] I have added unit and/or integration tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: Sources/Kafka/KafkaGroup.swift [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of swift-kafka-client project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import Crdkafka
+
+let uuu: Int = 0
```
Comment (dimlio) [inline Sources/Kafka/KafkaGroup.swift]: ?

File: Sources/Kafka/ForTesting/RDKafkaClient+Group.swift [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+            if rd_kafka_event_error(resultEvent) != RD_KAFKA_RESP_ERR_NO_ERROR {
+                let errorMessage = String(cString: rd_kafka_event_error_string(resultEvent))
+                throw KafkaError.rdKafkaError(wrapping: rd_kafka_event_error(resultEvent), errorMessage: errorMessage)
+            }
+
+            let result = rd_kafka_event_ListConsumerGroups_result(resultEvent)
+
+            var ret = [KafkaGroup]()
+            var groupsCnt: size_t = 0
+            let groups = rd_kafka_ListConsumerGroups_result_valid(result, &groupsCnt)
```
Comment (dimlio) [inline Sources/Kafka/ForTesting/RDKafkaClient+Group.swift:58]:
```suggestion
            guard let groups = rd_kafka_ListConsumerGroups_result_valid(result, &groupsCnt) else {
                return []
            }
```

**Review by dimlio — APPROVED** [review]

File: Sources/Kafka/KafkaGroup.swift (hunk 1 above)
Comment (ser-0xff) [reply Sources/Kafka/KafkaGroup.swift]: Fixed.

File: Sources/Kafka/ForTesting/RDKafkaClient+Group.swift (hunk 2 above)
Comment (ser-0xff) [reply Sources/Kafka/ForTesting/RDKafkaClient+Group.swift:58]: Fixed.

**Review by blindspotbounty — APPROVED** [review]

**Pull request merged**